app/views/admin/services/forms/_categories.html.haml

```diff
@@ -7,4 +7,9 @@
 
   = hidden_field_tag 'service[category_ids][]', nil
   = field_set_tag nil, id: 'categories' do
-    = nested_categories(Category.arrange(order: :taxonomy_id))
+    = f.label :services
+    = nested_categories(Category.where(type: 'service').arrange(order: :taxonomy_id))
```

Comment:
could add scopes to Category model, e.g.:
```
scope :service, -> { where(type: 'service') }
```
Then you can do `Category.services.arrange...`